components/global-navigation.md

```diff
@@ -397,3 +397,98 @@
 			</div>
 		</div>
 	</div>
+
+	<h5>Search suggestions</h5>
+	<div class="wds-global-navigation">
+		<div class="wds-global-navigation__content-bar row">
+			<a href="#" class="wds-global-navigation__logo col-sm-4 col-md-3 col-lg-3">
+				<svg class="wds-global-navigation__logo-fandom" alt="Fandom">
+					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-fandom"></use>
+				</svg>
+				<span class="wds-global-navigation__logo-powered-by">powered by</span>
+				<svg class="wds-global-navigation__logo-wikia" alt="Wikia">
+					<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-company-logo-wikia"></use>
+				</svg>
+			</a>
+			<div class="col-sm-5 col-md-6 col-lg-6 wds-global-navigation__middle-column wds-search-is-active">
+				<div class="wds-global-navigation__content-item wds-dropdown">
+					<div class="wds-dropdown-toggle wds-global-navigation__dropdown-toggle">
+						<span>Wikis</span>
+						<svg class="wds-icon wds-icon-tiny wds-dropdown-toggle-chevron" alt="user dropdown">
+							<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-icons-dropdown-tiny"></use>
+						</svg>
+					</div>
+					<div class="wds-dropdown-content wds-global-navigation__dropdown">
+						<ul class="wds-list">
+							<li><a href="#" class="wds-global-navigation__dropdown-link">Popularne wiki</a></li>
+							<li><a href="#" class="wds-global-navigation__dropdown-link">Centrum Społeczności</a></li>
+						</ul>
+					</div>
+				</div>
+				<form class="wds-global-navigation__search">
+					<div class="wds-dropdown wds-is-active wds-global-navigation__search-input-wrapper">
+						<label class="wds-dropdown-toggle wds-global-navigation__search-label">
+							<svg class="wds-icon wds-icon-small" alt="Search">
+								<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-icons-magnifying-glass"></use>
+							</svg>
+							<input class="wds-global-navigation__search-input" name="search" placeholder="Szukaj na Wookieepedia..." value="Star"/>
+						</label>
+						<button class="wds-button wds-is-text wds-global-navigation__search-close" type="button">
+							<svg class="wds-icon wds-icon-small wds-global-navigation__search-close-icon" alt="Close">
+								<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-icons-cross"></use>
+							</svg>
+						</button>
+						<div class="wds-dropdown-content wds-global-navigation__search-suggestions">
+							<ul class="wds-list">
+								<li class="wds-global-navigation__search-suggestion">
```

Comment:
I remember @tomasznapieralski decided to not add class name to every element in a list few days ago. Can we make sure we are consistent in what we do?